config/scheduler/config.yaml

```diff
@@ -9,6 +9,15 @@ internedStringsCacheSize: 100000
 metrics:
   port: 9000
   refreshInterval: 30s
+  schedulerMetrics:
+    scheduleCycleTime:
+      start: 1.0
+      factor: 1.1
+      count: 110
+    reconcileCycleTime:
+      start: 1.0
+      factor: 1.1
+      count: 110
```

Comment:
```suggestion
  metrics:
    scheduleCycleTimeHistogramSettings:
      start: 1.0
      factor: 1.1
      count: 110
    reconcileCycleTimeHistogramSettings:
      start: 1.0
      factor: 1.1
      count: 110
```

Reply:
Scheduler is implicit in `schedulerMetrics`. I'd use the more descriptive `schedulerCycleTimeHistogramSettings` since otherwise it's unclear what the settings are for.